Index: exercise.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/exercise.py b/exercise.py
new file mode 100644
--- /dev/null	(date 1646830088336)
+++ b/exercise.py	(date 1646830088336)
@@ -0,0 +1,5 @@
+while num==100
+    print('test again')
+    num= int (input("please enter number"))
+else:
+    print("welcome king")
\ No newline at end of file
